src/Compilers/CSharp/Test/Symbol/Compilation/TypeInfoTests.cs

```diff
@@ -22,19 +22,17 @@ public void Equality()
             EqualityTesting.AssertEqual(new TypeInfo(obj, int32, nullable, notNullable),
                                         new TypeInfo(obj, int32, nullable, notNullable));
 
-#pragma warning disable IDE0055 // Fix formatting: spacing is intentional to allow for visual field comparison
-            EqualityTesting.AssertNotEqual(new TypeInfo(obj, obj,   nullable, nullable),
+            EqualityTesting.AssertNotEqual(new TypeInfo(obj, obj, nullable, nullable),
                                            new TypeInfo(obj, int32, nullable, nullable));
 
             EqualityTesting.AssertNotEqual(new TypeInfo(int32, obj, nullable, nullable),
-                                           new TypeInfo(obj,   obj, nullable, nullable));
+                                           new TypeInfo(obj, obj, nullable, nullable));
 
-            EqualityTesting.AssertNotEqual(new TypeInfo(obj, int32, nullable,    nullable),
+            EqualityTesting.AssertNotEqual(new TypeInfo(obj, int32, nullable, nullable),
                                            new TypeInfo(obj, int32, notNullable, nullable));
 
             EqualityTesting.AssertNotEqual(new TypeInfo(obj, int32, nullable, nullable),
                                            new TypeInfo(obj, int32, nullable, notNullable));
-#pragma warning restore IDE0055 // Fix formatting
```

Comment:
@333fred it is not possible to suppress Format Document within a region of a document. The warning suppression here was ignored since the code inside it was automatically reformatted anyway.

Reply:
➡️ See discussion in https://github.com/dotnet/roslyn/pull/38562#discussion_r321929178